packages/jest-circus-allure-reporter/.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+const nodeConfig = require( '@automattic/calypso-eslint-overrides/node' );
+
+module.exports = {
+	...nodeConfig,
+	rules: {
+		...nodeConfig.rules,
+
+		'require-jsdoc': [
+			'error',
+			{
+				require: {
+					FunctionDeclaration: true,
+					MethodDefinition: true,
+					ClassDeclaration: true,
+					ArrowFunctionExpression: false,
+					FunctionExpression: false,
+				},
+			},
+		],
+		'@typescript-eslint/no-unused-vars': [
+			'error',
+			{ argsIgnorePattern: '^_', ignoreRestSiblings: true },
+		],
+		'import/no-nodejs-modules': 'off',
```

Comment:
This is copied from the `@automattic/calypso-e2e` package but with one additional change at this line, as the built-in package `crypto` and `os` are used.

Reply:
Interesting -- I thought the "nodeConfig" would allow nodeJS modules. (After all, the whole point of that module is to to share the config for any node projects which need it)